src/components/OrderPanel/index.tsx

```diff
@@ -25,7 +26,7 @@ const OrderPanel: React.SFC<OrderPanelProps> = ({ sellToken, buyToken }) => (
     <AuctionPriceBar header="Closing Price" />
     <AuctionSellingGetting />
     {/* TODO: replace onclick with some logic (maybe: "to" prop) */}
-    <ButtonCTA onClick={() => console.log('Continuing to wallet')} to="/wallet">
+    <ButtonCTA onClick={() => console.log('Continuing to wallet')} to={+sellAmount > 0 && '/wallet'}>
```

Comment:
You know what would be nice for indication that we can't go further, `disabled={sellAmount>0}`. Then pass this sell amount all the way through to `<a disabled>` and change its style with `a.class[disabled]`.

Reply:
Also I'm not sure but when `<Link to={not location object or string}/>` there may be an error when you click it.

Reply:
OK cool - good idea. Will dig into this.